update documentation and tweak discord notification for better accuracy (#124)
microsoft/design-to-code: PUBLISHING.md

5. Always set `NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}` to minimize authentication issues. This will automatically create the correct `.npmrc` configuration for the workflow.

## GIT Authentication Best Practices

> 1. Automated committing to a protected branch such as `main` requires a PAT (Personal Access Token) from a user with 'workflow' and 'write:packages' security permissions.

Reviewer: ```suggestion
1. Automated committing to a protected branch such as `main` requires a PAT (Personal Access Token) from a user with 'git' security permissions specifying 'repo' during the creation of the PAT.
```

Reviewer: I used a PAT on my account, you can't reuse a PAT from somewhere else as it will not show it to you even if you have permissions.
